### PR TITLE
Added board AirM2M CORE ESP32C3

### PR DESCRIPTION
**Pull request by xe5700**
## Description:
Added board AirM2M CORE ESP32C3

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [x] I accept the [DCO](https://github.com/1technophile/OpenMQTTGateway/blob/development/docs/participate/development.md#developer-certificate-of-origin).

### CODE REVIEW

File: environments.ini [hunk 1]
```diff
@@ -1483,6 +1483,23 @@ build_flags =
...
+  ${libraries.ble}
+  ${libraries.decoder}
+build_flags =
+  ${com-esp.build_flags}
+  '-DZgatewayBT="BT"'
+  '-DLED_INFO=19'
+  '-DLED_SEND_RECEIVE=20'
+  '-DNO_INT_TEMP_READING=true' ; Internal temperature reading not building on ESP32 C3 or S3
+  '-DGateway_Name="OpenMQTTGateway_AirM2M_Core_ESP32C3"'
+
```
Comment (1technophile) [inline environments.ini:1504]: Could you add a `custom_description` please

**Comment by xe5700** [comment]
I added it

File: environments.ini (hunk 1 above)
Comment (xe5700) [reply environments.ini:1504]: I added it

Comment (1technophile) [reply environments.ini:1504]: thanks

**Comment by 1technophile** [comment]
Thanks

**Pull request merged**